api/playdate/Cargo.toml

```diff
@@ -17,6 +17,7 @@ ctrl = { workspace = true, default-features = false }
 display = { workspace = true, default-features = false }
 fs = { workspace = true, default-features = false }
 gfx = { workspace = true, default-features = false }
+lua = { workspace = true, default-features = false }
 menu = { workspace = true, default-features = false }
 scoreboards = { workspace = true, default-features = false }
```

Comment:
Should I update version numbers as part of this pull request?

Reply:
Optionally, but could be cool. If not, I'll make follow up with it.
But merge will trigger the publish workflow and it will failed I you do not update version.

Reply:
_So strange that [workflow `publish`](https://github.com/boozook/playdate/actions/runs/8965082458/job/24617906492#step:6:3) (with `dry-run`, don't be afraid) used as check "is changed crates bumped?" passes `playdate` crate 🤷🏻‍♂️_. It had to fail because crate updated, but version is not. I've created issue for this - https://github.com/katyo/publish-crates/issues/743 .

Reply:
Anyway, could you please set version of `playdate` package to `0.2.0`?.

Reply:
Fixed in https://github.com/paulyoung/playdate/pull/2/commits/4f3e324aaaee1bcff2f2caf622bf5208e4890469